### ISSUE TITLE
Select Particular index on click

### ISSUE THREAD

**Issue by pixelbendr** [aaronbond/SwipeDeck2#1]
So I have a list of profiles in an Array List and I want the user to be able to start swiping from anywhere in the list. is there a method for initialising this approach or another event like cardStart or cardEnd?

**Comment by aaronbond** [comment]
do you mean you want the deck to start from anywhere in the adapter?

could you explain what you want to acheive? what do you want to happen when you tell swipedeck to start at another point in the adapter and when i understand exactly what you're after i'll implement it.

**Comment by pixelbendr** [comment]
Yes please. I have a list of user profiles that am swiping through and I want swipe deck to handle the indexing so when I click on profile 5 swiping right goes to profile 6 and swiping left goes to profile 4.

**Comment by aaronbond** [comment]
I'm still not exactly sure what you're after, so let me get this clear.

let's say you have 10 cards:
1 2 3 4 5 6 7 8 9 0
            ^
you want to, for example, start the deck at card 5, and swipe left goes forward a card and right backwards a card? 

if that's the case, do you want it to be animated? how do you imagine it to look to the user?

**Comment by geraldjaydlagarto** [comment]
@aaronbond Greetings, I am using your library to our project. Still hoping for a reverse function or load back the previous card function :)
